src/lib/components/list-item/stories/index.stories.tsx

```diff
@@ -119,10 +119,10 @@ export const WithPrefixAndSuffixRadioWithoutDescription: Story = {
   },
 };
 
-export const WithPrefixAndSuffixSwitchWithoutDescription: Story = {
+export const WithPrefixAndSuffixToggleWithoutDescription: Story = {
```

Comment:
`CombinedToggle` instead `WithPrefixAndSuffixToggleWithoutDescription`

Reply:
> `CombinedToggle` instead `WithPrefixAndSuffixToggleWithoutDescription`

Sir, this is not my task, but I'll take this